routes/users.js

```diff
@@ -107,15 +107,14 @@ router.get('/login', redirectProfile, (req, res) => {
 router.get('/profile', redirectLogin, async (req, res) => {
   try {
     let user = req.session.user;
-    let params = { user };
     if (user.userRole === 'partner') {
       let agency = await Agency.findOne({ accountManager: user._id });
+      // If user hadn't filled out agency info, redirect them to form
       if (!agency) {
-        return res.render('agency');
+        return res.status(200).render('agency');
```

Comment:
isnt status 200 default? just out of interest

Reply:
It is; I think I was testing something and added that iirc, not that it makes any difference anyways lol.